android/src/main/java/io/mosip/tuvali/wallet/transfer/TransferHandler.kt

```diff
@@ -47,8 +47,9 @@ class TransferHandler(looper: Looper, private val central: Central, val serviceU
       IMessage.TransferMessageTypes.INIT_RESPONSE_TRANSFER.ordinal -> {
         val initResponseTransferMessage = msg.obj as InitResponseTransferMessage
         val responseData = initResponseTransferMessage.data
+        val maxChunkSize = initResponseTransferMessage.maxDataBytes
         Log.d(logTag, "Total response size of data ${responseData.size}")
-        chunker = Chunker(responseData)
+        chunker = Chunker(responseData, maxChunkSize)
```

Comment:
Maybe sometime later, it'd be worthwhile to change ChunkerBase's parameter's name in it's constructor(from ChunkerBase.kt & Chunker.kt) from `mtuSize` to `maxChunkBytes` or something similar & more appropriate. Right now, it takes the MTU but it doesn't mean the ATT MTU(the maximum bytes one can write plus the packet header).